app/assets/stylesheets/farmbot.css.scss

```diff
@@ -98,6 +100,10 @@ $brown:          #CA8;
   }
 }
 
+.plantCatalogTile{
```

Comment:
Selector `plantCatalogTile` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space